src/libraries/Microsoft.Extensions.Hosting/ref/Microsoft.Extensions.Hosting.cs

```diff
@@ -32,14 +32,16 @@ public static partial class Host
         public static Microsoft.Extensions.Hosting.IHostBuilder CreateDefaultBuilder(string[]? args) { throw null; }
         public static Microsoft.Extensions.Hosting.HostApplicationBuilder CreateEmptyApplicationBuilder(Microsoft.Extensions.Hosting.HostApplicationBuilderSettings? settings) { throw null; }
     }
-    public sealed partial class HostApplicationBuilder
+    public sealed partial class HostApplicationBuilder : Microsoft.Extensions.Hosting.IHostApplicationBuilder
     {
         public HostApplicationBuilder() { }
         public HostApplicationBuilder(Microsoft.Extensions.Hosting.HostApplicationBuilderSettings? settings) { }
         public HostApplicationBuilder(string[]? args) { }
         public Microsoft.Extensions.Configuration.ConfigurationManager Configuration { get { throw null; } }
         public Microsoft.Extensions.Hosting.IHostEnvironment Environment { get { throw null; } }
         public Microsoft.Extensions.Logging.ILoggingBuilder Logging { get { throw null; } }
+        Microsoft.Extensions.Configuration.IConfigurationManager Microsoft.Extensions.Hosting.IHostApplicationBuilder.Configuration { get { throw null; } }
+        System.Collections.Generic.IDictionary<object, object> Microsoft.Extensions.Hosting.IHostApplicationBuilder.Properties { get { throw null; } }
```

Comment:
>object [](http://example.com/codeflow?start=55&length=6)

Just curious, nullable values are not valid?

Reply:
Correct. This matches:

https://github.com/dotnet/runtime/blob/3eb4407e441b539c81c6ea8cc797f97536abd01f/src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/IHostBuilder.cs#L14-L19

https://github.com/dotnet/runtime/blob/3eb4407e441b539c81c6ea8cc797f97536abd01f/src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/HostBuilderContext.cs#L36-L39